tasks/main.yml

```diff
@@ -19,6 +27,10 @@
   become: '{{ conda_env_escalate }}'
   become_user: root
   command: '{{ conda_env_bin_dir }}/conda env {{ conda_env_action_command }} -f={{ conda_env_fq_dest_environment }} -q -n {{ conda_env_name }}'
+  args:
+    # pretend task creates nothing to always run when updating an environment. In general, the first directory
+    # is where conda creates new environments.
+    creates: "{% if conda_env_action_command == 'update' %}{% else %}{{ conda_info.envs_dirs | first }}/{{ conda_env_name }}{% endif %}"
```

Comment:
I don't understand the `first` bit and I don't want to delay your progress.

Reply:
`conda` has a list of directories where it will look for environments. By default that's the `envs` directory in the install directory but a second one is `~/.conda/envs`.

Since the create environment task doesn't use a specific prefix it will use the `first` one in that list.